nohup: ignoring input
Saving to  /home/011/e/ec/ecj200000/CS4375Project/simple_steel_training_results
Starting lr 1e-05
Starting epoch 1: 05.08.23-09.15
Traceback (most recent call last):
  File "/usr/lib64/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/usr/lib64/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/home/011/e/ec/ecj200000/CS4375Project/train_steel_simplefcn.py", line 70, in <module>
    pred_labels = network(imgs)
  File "/home/011/e/ec/ecj200000/.local/lib/python3.6/site-packages/torch/nn/modules/module.py", line 1102, in _call_impl
    return forward_call(*input, **kwargs)
  File "/home/011/e/ec/ecj200000/CS4375Project/fcnn.py", line 26, in forward
    x = ds_layer(x)
  File "/home/011/e/ec/ecj200000/.local/lib/python3.6/site-packages/torch/nn/modules/module.py", line 1102, in _call_impl
    return forward_call(*input, **kwargs)
  File "/home/011/e/ec/ecj200000/.local/lib/python3.6/site-packages/torch/nn/modules/container.py", line 141, in forward
    input = module(input)
  File "/home/011/e/ec/ecj200000/.local/lib/python3.6/site-packages/torch/nn/modules/module.py", line 1102, in _call_impl
    return forward_call(*input, **kwargs)
  File "/home/011/e/ec/ecj200000/.local/lib/python3.6/site-packages/torch/nn/modules/batchnorm.py", line 179, in forward
    self.eps,
  File "/home/011/e/ec/ecj200000/.local/lib/python3.6/site-packages/torch/nn/functional.py", line 2283, in batch_norm
    input, weight, bias, running_mean, running_var, training, momentum, eps, torch.backends.cudnn.enabled
RuntimeError: CUDA out of memory. Tried to allocate 704.00 MiB (GPU 0; 15.89 GiB total capacity; 1.05 GiB already allocated; 357.88 MiB free; 1.06 GiB reserved in total by PyTorch) If reserved memory is >> allocated memory try setting max_split_size_mb to avoid fragmentation.  See documentation for Memory Management and PYTORCH_CUDA_ALLOC_CONF
